packages/theme_tailor_annotation/lib/src/theme_tailor.dart

```diff
@@ -15,6 +15,7 @@ class Tailor {
     this.themeGetter = ThemeGetter.onBuildContextProps,
     this.encoders,
     this.requireStaticConst = false,
+    this.generateStaticGetters = true,
```

Comment:
I think it should be only true for the dev version while working on changes regarding themes therefore by default it should be false.
Preferably someone can set it to true from the build.yaml if needed